test/Test/BaseDAO/Proposal.hs

```diff
@@ -195,6 +194,27 @@ insufficientTokenVote = uncapsNettest $ do
   callFrom (AddressResolved owner2) dao (Call @"Vote") params
     & expectCustomError_ #vOTING_INSUFFICIENT_BALANCE
 
+voteOutdatedProposal :: (Monad m) => NettestImpl m -> m ()
+voteOutdatedProposal = uncapsNettest $ do
+  ((owner1, _), (owner2, _), dao, admin) <- originateBaseDaoWithConfig config
+
+  -- | Create sample proposal
+  key1 <- createSampleProposal 1 owner1 dao
+
+  callFrom (AddressResolved admin) dao (Call @"Set_voting_period") 20
+
+  let params = DAO.VoteParam
+        { vVoteType = True
+        , vVoteAmount = 2
+        , vProposalKey = (key1 :: ByteString)
```

Comment:
Is it necessary to specify the type explicitly here?

Reply:
It's not, I've removed this, Ivan.